src/assets/css/main.scss

```diff
@@ -0,0 +1,99 @@
+  @font-face {
+    font-family: "november";
+    font-style: normal;
+    font-weight: 500;
+    font-display: auto;
+    src: url("../fonts/november.ttf");
+  }
+
+  @font-face {
+    font-family: "AVF";
+    font-style: light;
+    font-weight: 300;
+    font-display: auto;
+    src: url("../fonts/AVF.ttf");
+  }
+
+//   background: linear-gradient(0deg, #F1DE00, #F1DE00),
+// linear-gradient(0deg, #000000, #000000),
```

Comment:
Hey, can you please remove these commented out lines?

Reply:
sure! :) i left them for raffa so he would know how I did the circle that we talked about. but as he as forked this anyway, i am happy to delete :)